thrust/thrust/system/cpp/detail/execution_policy.h

```diff
@@ -25,63 +11,104 @@
 #elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
 #  pragma system_header
 #endif // no system header
+
+#include <thrust/detail/allocator_aware_execution_policy.h>
+#include <thrust/system/cpp/detail/execution_policy.h>
 #include <thrust/system/detail/sequential/execution_policy.h>
 
 THRUST_NAMESPACE_BEGIN
-// put the canonical tag in the same ns as the backend's entry points
-namespace system::cpp
-{
-namespace detail
+namespace cpp
```

Comment:
Critical: that's probably a breaking change if users forward declare backend specific execution policies.